src/erk/artifacts/sync.py

```diff
@@ -0,0 +1,113 @@
+"""Artifact synchronization from erk package to project."""
+
+from dataclasses import dataclass
+from pathlib import Path
+
+from erk.artifacts.models import ArtifactState
+from erk.artifacts.staleness import get_current_version
+from erk.artifacts.state import save_artifact_state
+from erk.kits.hooks.installer import install_hooks
+from erk.kits.io.manifest import load_kit_manifest
+from erk.kits.models.artifact import ARTIFACT_TARGET_DIRS, ArtifactType
+from erk.kits.operations.artifact_operations import create_artifact_operations
+
+
+@dataclass
+class SyncResult:
+    """Result of artifact sync operation."""
+
+    success: bool
+    artifacts_installed: int
+    hooks_installed: int
+    error: str | None = None
+
+
+def sync_artifacts(project_dir: Path) -> SyncResult:
+    """Sync artifacts from erk package to project.
+
+    This function copies artifacts from the bundled erk kit to the project
+    directory and installs any associated hooks.
+
+    Args:
+        project_dir: Project root directory
+
+    Returns:
+        SyncResult with counts of installed artifacts and hooks
+    """
+    # Inline import to avoid import-time side effects
+    from erk_kits import get_kits_dir
```

Comment:
make this top-level

Reply:
Resolved via /erk:pr-address at 2025-12-29 09:45 PM PST - Moved get_kits_dir import to module-level

_Addressed via `/erk:pr-address` at 2025-12-29 21:45 PST_